server/jetstream.go

```diff
@@ -1421,11 +1440,12 @@ func (js *jetStream) sufficientResources(limits *JetStreamAccountLimits) error {
 	if limits == nil {
 		return nil
 	}
+
 	if js.memReserved+limits.MaxMemory > js.config.MaxMemory {
-		return fmt.Errorf("insufficient memory resources available")
+		return ErrMemoryResourcesExceeded
```

Comment:
dont we have to handle -1 MaxStore and MaxMemory here too?

Reply:
js will have values, and when limits.Max are -1 its a no-op essentially. So not IMO.